scripts/test/runtests.sh

```diff
@@ -81,4 +85,7 @@ done
 popd
 set -e
 
+#resetting the path so that we can be good neighbors to other scripts running after this.
+export PATH=$StartPath
```

Comment:
NO need for this in bash. Path will be local to this subshell and child shells only.